docs/source/install/system_requirements.rst

```diff
@@ -10,6 +10,8 @@ testing. Yes, using exactly the same boxes will improve your experience.
 +===================+==============================================================================+===================================================================================================================================================================+
 | Ubuntu 14.04      | `bento/ubuntu-14.04 <https://atlas.hashicorp.com/bento/boxes/ubuntu-14.04>`_ | `Ubuntu Server 14.04 LTS (HVM)  <https://aws.amazon.com/marketplace/pp/B00JV9TBA6/ref=srh_res_product_title?ie=UTF8&sr=0-3&qid=1457037882965>`_                   |
 +-------------------+------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Ubuntu 16.04      | `bento/ubuntu-16.04 <https://atlas.hashicorp.com/bento/boxes/ubuntu-16.04>`_ | `Ubuntu 16.04 LTS - Xenial (HVM)  <https://aws.amazon.com/marketplace/pp/B01JBL2M0O/>`_                                                                           |
```

Comment:
Yeah, it's Vagrant.
In some version ubuntu/xenial works but bento16 is broken, in another Vagrant version bento16 works and ubuntu/xenial is broken.

So this bento will work for some people.